Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -168,6 +170,13 @@ define([
          * @private
          */
         this._clippingPlanes = undefined;
+
+        /**
+         * A property specifying a {@link Rectangle} used to selectively prevent tiles outside a region from loading.
+         * For limiting terrain in scenes that use custom projections or Proj4JS projections that cause overlapping tiles.
```

Comment:
I would remove the line about custom projections since this would likely get merged before support is added for them and that is not the only use case.

Prevents tiles from loading and also clips tiles on the boundary?